tests/baselines/reference/privateNameNestedMethodAccess.errors.txt

```diff
@@ -0,0 +1,26 @@
+tests/cases/conformance/classes/members/privateNames/privateNameNestedMethodAccess.ts(11,25): error TS18014: The property '#bar' cannot be accessed on type 'C' within this class because it is shadowed by another private identifier with the same spelling.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameNestedMethodAccess.ts (1 errors) ====
+    const C = class {
+        #foo = 42;
+        #bar() { new C().#baz; }
+        get #baz() { return 42; }
+    
+        m() {
+            return class D {
+                #bar() {}
+                constructor() {
+                    new C().#foo;

```

Comment:
Maybe add a test with `any` to check that the private access is correct in the nested scope even without type info ?